executor/aggregate_test.go

```diff
@@ -306,97 +306,6 @@ func (s *testSuite) TestAggregation(c *C) {
 	tk.MustQuery("select sum(a), sum(b) from idx_agg where b > 0 and b < 10")
 }
 
-func (s *testSuite) TestStreamAgg(c *C) {
```

Comment:
This test can be covered by others. And it's inconvenient to export everything in baseExecutor.